command/agent/agent.go

```diff
@@ -1039,12 +1039,17 @@ func (a *Agent) EnableServiceMaintenance(serviceID string) error {
 		return nil
 	}
 
+	// Use default notes if no reason provided
+	if reason == "" {
+		reason = "Maintenance mode is enabled for this service"
```

Comment:
Agreed on the constant. Also a recommendation to make this a bit more verbose and add verbiage such as: "No specific reason was provided. This is a default message."